solutions/usaco-831.mdx

```diff
@@ -229,11 +295,47 @@ public class TeamTicTacToe {
 	}
 	
 	//CodeSnip{Kattio}
-	
 }
 
 ```
 
 </JavaSection>
+<PySection>
+
+```py
+WIDTH = 3
+
+
+def cow_contrib(game: [str], pts: [(int, int)]) -> {str}:
+	contained = set()
+	for pt in pts:
+		contained.add(game[pt[0]][pt[1]])
+	return contained
+
+
+with open('tttt.in') as read:
+	board = [read.readline() for _ in range(WIDTH)]
+
+winners = [set() for _ in range(WIDTH + 1)]
+# A function that adds the amount of cows who contributed to the winners count
```

Comment:
hmm i think this comment is kind of confusing, though i'm not sure how to rephrase it?

it's more like "A function that updates the winners array with a winning team of cows" but that's also not a good description... does anyone have any ideas?

Reply:
"function which adds a winning team to the winners array"?

or how about instead of adding the pairs, you just keep a counter, that might be easier to understand (?)

Reply:
i'll let you two decide

Reply:
either is ok.